internal/handlers/schedule_backup.go

```diff
@@ -0,0 +1,53 @@
+package handlers
+
+import (
+	"context"
+	"time"
+	"ydbcp/internal/connectors/db/yql/queries"
+	"ydbcp/internal/server/services/backup"
```

Comment:
OperationHandler should not depend on grpc handler. I suggested to move the common code to an independent package

Reply:
needed symbols moved to ```ydbcp/internal/impl```